### PR TITLE
Mark Yen

### PR DESCRIPTION
**Pull request by markcyen**
Tried my best! Taking to long to think... yikes!

### CODE REVIEW

File: lib/item.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Item
+  attr_reader :name,
+              :price
+
+  def initialize(name:, price:)
```
Comment (mikedao) [inline lib/item.rb:5]: Intialize should only take one argument the hash.

File: lib/event.rb [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    @food_trucks << food_truck
+  end
+
+  def food_truck_names
+    @food_trucks.map do |food_truck|
+      food_truck.name
+    end
+  end
+
+  def food_truck_that_sell(item)
```
Comment (mikedao) [inline lib/event.rb:20]: This method name does not match the specification.

**Review by mikedao — COMMENTED** [review]
What you have here is good. Make sure you are following the instructions carefully. Would liked to have seen you completed more of the challenge.

Score: 2.0

File: lib/item.rb (hunk 1 above)
Comment (markcyen) [reply lib/item.rb:5]: @mikedao, is this a stylistic comment? Seems like you could initialize a hash a number of ways? https://thoughtbot.com/upcase/videos/ruby-keyword-arguments (see section on "Refactoring with keyword arguments")

File: lib/event.rb (hunk 2 above)
Comment (markcyen) [reply lib/event.rb:20]: @mikedao, just want to make sure why I did this incorrectly. The return value  is an array of the food truck and I thought  find_all returns an array? My test reflects the array of food trucks that only includes the specific item.

pry(main)> event.food_trucks_that_sell(item1)
#=> [#<FoodTruck:0x00007fe1348a1160...>, #<FoodTruck:0x00007fe134910650...>]

pry(main)> event.food_trucks_that_sell(item4)
#=> [#<FoodTruck:0x00007fe1349bed40...>]

Comment (markcyen) [reply lib/event.rb:20]: Nevermind on this one... I just had an incorrect spelling.

**Pull request closed without merging**